doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst

```diff
@@ -145,7 +143,7 @@ When using logs, this method has the following advantages:
 Configuration
 -------------
 
-.. to review
+.. TO_REVIEW
 
 To enable the assisted multicore logging for a domain, set the :kconfig:option:`CONFIG_LOG_STM_DICT` Kconfig option for that domain's image to ``y``.
```

Comment:
It has changed now. Logging is enabled using a snipped `nordic-log-stm` or `nordic-log-stm-dict` for dictionary-based logging.

Reply:
I confirm, the logging works with these 2 snippets

Reply:
@PavelVPV @nordic-krch How to enable these snippets?

Reply:
through `-S` option passed to west build command, for example:
```
west build -b nrf54h20dk/nrf54h20/cpuapp -S nordic-log-stm
```